aggregator/src/aggregator.rs

```diff
@@ -117,6 +117,16 @@ pub mod report_writer;
 #[cfg(test)]
 mod taskprov_tests;
 
+pub(crate) fn aggregation_success_counter(meter: &Meter) -> Counter<u64> {
```

Comment:
nit/suggestion: we should avoid growing this file if we can. `janus_aggregator::metrics` seems like a more natural place for this item, if it's going to be `pub(crate)` anyway. `aggregate_step_failure_counter` could be moved there too, but perhaps not in this PR.